migrator/clone/postgres/db_clone_manager_impl.go

```diff
@@ -122,8 +122,6 @@ func (d *dbCloneManagerImpl) Scan() error {
 		log.Infof("clone %s is of version %v", PreviousClone, ver)
 
 		d.cloneMap[PreviousClone] = metadata.NewPostgres(ver, PreviousClone)
-	} else {
```

Comment:
This was causing issue rolling back after a restart because it would delete previous.  Since dodging that would cause a fair amount of logic, I simply won't delete the `central_previous`.